entrypoint.sh

```diff
@@ -37,7 +39,7 @@ fi
 
 mount --make-shared /sys/fs/cgroup
 mount --make-shared /proc
-mount --make-shared /var/gitpod
+mount --make-shared /var/gitpod/workspaces
```

Comment:
This was done intentionally, as I wanted to output the root cert into `/var/gitpod/gitpod-ca.crt`  and `workspaces` folder felt specific to workspace (updated README around this) WDYT?